src/abhijitMJun20/Staticnonstatic.java

```diff
@@ -0,0 +1,56 @@
+package abhijitMJun20;
+
+public class Staticnonstatic { 
+	static double sumofvalues;
```

Comment:
Abhijit, have you read assignment properly.
Rule D) Class should not have any instance variable. Remove instance variables please.

Reply:
Done